portfolio/src/main/webapp/script.js

```diff
@@ -72,3 +41,32 @@ function addRandomFact() {
     const greetingContainer = document.getElementById('greeting-container');
     greetingContainer.innerText = response;
 }
+
+/** Fetch and display the desired number of comments to the portfolio. */
+async function displayComments() {
+    const numComments = document.getElementById('num-comments').value;
+    const response = await fetch('/data?num-comments=' + numComments);
+    const messageArr = await response.json();
+
+    // Split messageArr into paragraph elements (maybe splice)
+    var output = messageArr.map(entity => "<p>" + entity.comment + "</p>");
+    document.getElementById('comment_section').innerHTML = output.join("");
+}
+
+/** Check if user entered the correct password. */
```

Comment:
Maybe add a comment here about what we discussed earlier (that this is not a secure password storing mechanism, but that it's good as a learning experience for now!)

Reply:
OK